try:
    print(num)
except Exception as result:
    print(result)#name 'num' is not defined